pydc_client.py

```diff
@@ -381,7 +381,7 @@ def server_handler(self,data,info,args): # Interacts with the DC, responding to
 						if nick=="": continue
 						self._nicklist[nick]["bot"] = (True if nick in bots else False)
 				elif x[0]=="$MyINFO":
-					nick,desc,conn,flag,email,share = re.findall("^\$MyINFO \$ALL ([^ ]*) ([^\$]*)\$ \$([^\$]*)([^\$])\$([^\$]*)\$([^\$]*)\$$",data)[0]
+					nick,desc,conn,flag,email,share = re.findall("^\$MyINFO \$ALL ([^ ]*) ([^\$]*)\$.\$([^\$]*)([^\$])\$([^\$]*)\$([^\$]*)\$",data)[0]
```

Comment:
Kept pressing the up arrow key in ipython and finally found the string.
`$MyINFO $ALL bourne99 <ApexDC++ V:1.6.0,M:P,H:1/0/0,S:3>$P$0.11$$125525478604$`